src/app.types.tsx

```diff
@@ -68,25 +94,19 @@ export interface CatalogueItemPropertyResponse {
   unit: string;
 }
 
-export interface CatalogueItem {
+export interface CatalogueItem extends CatalogueItemDetails {
   catalogue_category_id: string;
-  name: string | undefined;
-  description: string;
   properties: CatalogueItemPropertyResponse[];
   manufacturer: CatalogueItemManufacturer;
   id: string;
 }
-export interface AddCatalogueItem {
+export interface AddCatalogueItem extends CatalogueItemDetails {
```

Comment:
Happy to leave it for now, but I think it makes more sense the optional fields here to be undefined instead of null as the backend will take care of the defaults, then EditCatalogueItem can extend `Partial<AddCatalogueItem>` to make the remaining fields optional. This would mean CatalogueItem couldn't inherit from the same thing, and CatalogueItemDetailsPlaceholder would need to be | undefined as well though.